code/game/objects/items/clown_items.dm

```diff
@@ -79,14 +79,23 @@
 	new /obj/effect/particle_effect/foam(loc)
 	return (TOXLOSS)
 
-/obj/item/soap/proc/decreaseUses(mob/user)
-	uses--
+/obj/item/soap/proc/decreaseUses(mob/user, amount = 1)
+	uses -= amount
 	if(uses <= 0)
 		to_chat(user, "<span class='warning'>[src] crumbles into tiny bits!</span>")
 		qdel(src)
 
 /obj/item/soap/afterattack(atom/target, mob/user, proximity)
 	. = ..()
+	if(iscarbon(target) && user == target && user.zone_selected == BODY_ZONE_PRECISE_MOUTH && user.a_intent == INTENT_HELP) //mmm, soap...
```

Comment:
No. No feeding soap to others. Deepfry the soap or whatever to do that, but it won't be toxic then. You only eat soap yourself to impress your fellow greytiders, not turn it into a weapon.

Reply:
> No. No feeding soap to others. Deepfry the soap or whatever to do that, but it won't be toxic then. You only eat soap yourself to impress your fellow greytiders, not turn it into a weapon.

Why not just make it non-poisonous then instead of introducing a magic forcefield that doesn't let you feed food to someone else. No other item has this restriction so you're creating a snowflake mechanic that only applies to your item.